site2/website/blog/2021-12-21-apache-pulsar-hits-10000-github-stars.md

```diff
@@ -0,0 +1,55 @@
+
+Pulsar was initially developed as a cloud-native distributed messaging system at Yahoo! in 2012. Over the past decade, Pulsar has evolved into a unified messaging and streaming platform for event-driven enterprises at scale. In this blog, we look at Pulsar’s community growth, project updates, ecosystem developments, and what’s next for the project. 
+
+# Community Growth
+
+Pulsar’s success depends on its community. As shown in the chart below, the growth of Pulsar’s GitHub stars accelerated after Pulsar became a top-level Apache Software Foundation project. 
+
+![](https://i.imgur.com/kwxz2XT.png)
+
+The number of contributors over time is another metric for measuring community engagement. The chart below shows that the number of Pulsar contributors accelerated when Pulsar became a top-level Apache project and the growth rate has continued into 2021. 
+
+![](https://i.imgur.com/TYy4CQg.png)
+
+[Pulsar surpassed Kafka in the number of monthly active contributors in early 2021](https://www.apiseven.com/en/contributor-graph?chart=contributorMonthlyActivity&repo=apache/pulsar,apache/kafka). This shows that the development and engagement of Pulsar has grown rapidly over the past few years. 
+
+![](https://i.imgur.com/6AN3dtM.png)
+
+# Project Updates and Ecosystem Development
+
+Recent Pulsar project updates have brought new capabilities to the project. Below we look at key releases and launches.  
+
+## 1. Pulsar 2.8 - Unified Messaging and Streaming with Transactions
+
+Pulsar 2.8 (released in June 2021) introduced many major updates. The Pulsar Transaction API was added to support atomicity across multiple topics and enable end-to-end exactly-once message delivery guarantee for streaming jobs. Replicated subscriptions was added to enhance Pulsar’s geo-replication. With this feature, a consumer can restart consuming from the failure point in a different cluster in case of failover. 
+
+## 2. Pulsar Connectors - AWS SQS Connector, Cloud Storage Sink Connector, and More
+
+Pulsar connectors enable easy integration between Pulsar and external systems with added benefits. For instance, the AWS SQS Connector enables secure integration between Pulsar and SQS without needing to write any code. And the Cloud Storage Sink Connector can export data by guaranteeing exactly-once delivery semantics to its consumers. It provides applications that export data from Pulsar, the benefits of fault tolerance, parallelism, elasticity, and much more. 
+
+## 3. Protocol Handlers 
+
+Protocol handlers, such as MQTT-on-Pulsar (MoP), allow Pulsar to interact with applications built on other messaging platforms, lowering the barrier to Pulsar adoption. MoP enables MQTT applications to leverage Pulsar’s infinite event stream retention with BookKeeper and tiered storage.
```

Comment:
I see you removed KoP because it's a project from a Pulsar vendor (StreamNative). But MoP is also a StreamNative project.